### ISSUE TITLE
Training Models?

### ISSUE THREAD

**Issue by coogle** [CodeWithKyrian/transformers-php#45]
### Your question

Does this library support Training of models? e.g. the `Trainer` class? I don't see any documentation for it. In our use case we'd like to train a model to label user input based on a known-set of approximately 200 potential labels.

**Comment by CodeWithKyrian** [comment]
Hi @coogle, unfortunately TransformersPHP doesn't support training at the moment, just inferencing as stated in the docs. Training or fine tuning has to be done using another stack like Pytorch or Tensorflow, and the model weights can now be converted to ONNX format afterwards to run inferencing in PHP
